`Funding a wallet` follow up PR (#604)
BitcoinDesign/Guide: guide/onboarding/funding-a-wallet.md

{% include /tip/open.html label="⚡ Lightning services" icon="info" color="blue" %}

Lightning services greatly simplify the UX of funding and using a Lightning wallet. We have highlighted the sections of funding a wallet with tip boxes like these that can benefit from using Lightning services.

> We have a dedicated page that covers funding a wallet using the outlined services.

Reviewer: We should add a link to this page here, I'm not sure which page is referred to.

Reviewer 2: Ah, this page is LSP page, it's in #568, so I assume Bosch will do a follow up once that one is finished, to avoid broken links.

{% include /tip/close.html %}

## Does the user own bitcoin?

The first step in funding a wallet is asking the user if they currently own any bitcoin.

> If your user does not own bitcoin, your wallet should guide them in obtaining some. Offering a guide on how to obtain bitcoin will prevent users from falling victim to scams and low quality exchanges. Alternatively, you could offer the option to buy directly within your wallet, which greatly simplifies the funding process.

Reviewer 3: ```suggestion
If your user does not own bitcoin, your wallet should provide practical tips to securely acquire bitcoin from trustworthy exchanges. Alternatively, you could offer the option to buy directly within your wallet, which greatly simplifies the funding process.
```
I think it's better to focus on the desired outcome and avoid the mention of scams.

   image = "/assets/images/guide/onboarding/funding-a-wallet/buy-bitcoin.png"
   retina = "/assets/images/guide/onboarding/funding-a-wallet/buy-bitcoin@2x.png"
   layout = "shadow"
   caption = "Alternatively, offer the option to buy bitcoin within your wallet."
   alt-text = "Screen showing option to buy within the wallet"
   width = 250
   height = 541
%}

</div>

## Funding the wallet

> Once the user has some bitcoin, the next step is for them to fund their wallet.

Reviewer 3: ```suggestion
Once the user has some bitcoin, the next step is to fund their wallet.
```
Shorter?

Maybe should add a note that this is only necessary if the user transfers bitcoin from another wallet. If they went the route of buying bitcoin through a service like Moonpay, they already had to enter an address to deposit to, which probably was one from this wallet.

## The wallet has been funded

Once the wallet is funded and a channel has been opened the user is ready to make payments.

Initially, the user will only have a payment channel with outbound liquidity meaning they will only be able to send via Lightning until they make some payments and have some inbound liquidity. This should be clearly communicated to the user with an explanation as to why.

{% include /tip/open.html label="⚡ On-demand channels" icon="info" color="blue" %}

> Offering on-demand channels as a Lightning service allows users to receive Lightning payment giving them both inbound and outbound liquidity.

Reviewer 3: ```suggestion
Offering on-demand channels as a Lightning service allows users to receive Lightning payments, giving them both inbound and outbound liquidity.
```
